modules/sdk-coin-sol/src/lib/ataInitializationBuilder.ts

```diff
@@ -13,17 +13,29 @@ import {
 } from './utils';
 import assert from 'assert';
 import { AtaInitializationTransaction } from './ataInitializationTransaction';
+import * as _ from 'lodash';
+
+export class TokenAssociateRecipient {
+  ownerAddress: string;
+  tokenName: string;
+}
+
+declare class DuplicateMethodError extends ExtendableError {
```

Comment:
LGTM. nit: Let's move this to `BitGoJS/modules/sdk-core/src/account-lib/baseCoin/errors.ts`